Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -37,23 +37,26 @@ public async Task<CreateAnywhereFleetResponse> CreateFleet(string fleetName)
         {
             if (_amazonGameLiftWrapper != null)
             {
+                if (string.IsNullOrWhiteSpace(fleetName))
+                {
+                    return Response.Fail(new CreateAnywhereFleetResponse
+                    {
+                        ErrorCode = ErrorCode.InvalidFleetName
+                    });
+                }
+                
                 var success = await CreateCustomLocationIfNotExists(FleetLocation);
                 if (success)
                 {
                     var fleetId = await CreateFleet(ComputeType.ANYWHERE, FleetLocation, fleetName);
-                    if (fleetId == null)
+                    if (!string.IsNullOrWhiteSpace(fleetId))
                     {
-                        return Response.Fail(new CreateAnywhereFleetResponse
+                        return Response.Ok(new CreateAnywhereFleetResponse()
                         {
-                            ErrorCode = ErrorCode.InvalidFleetName
+                            FleetId = fleetId,
+                            FleetName = fleetName
                         });
```

Comment:
If the fleetId is null you're really removing the correct ErrorCode with your latest changes

Reply:
Fixed with a better error code.